docs/reading-list.md

```diff
@@ -0,0 +1,164 @@
+# Reading list
+
+Here you can find a list of relevant resources, that you may find useful for
+learning new skills, or polishing the ones you already have.
+
+The topics cover a wide variety of topics, from basic Python coding to
+general developing guidelines.
+
+Some titles have been shortened for organization purposes.
+
```

Comment:
How about some statement like "If you know about any useful books, articles or videos you would like to see in this list, feel free to add them."